Code/GraphMol/new_canon.cpp

```diff
@@ -661,10 +652,7 @@ void initChiralCanonAtoms(const ROMol &mol,
 
 void freeCanonAtoms(std::vector<Canon::canon_atom> &atoms) {
   for (auto &atom : atoms) {
-    if (atom.nbrIds) {
-      free(atom.nbrIds);
-      atom.nbrIds = nullptr;
-    }
+    atom.nbrIds.reset();
```

Comment:
Are the atoms reused?  You might not need this with the unique_ptr

Reply:
good point